src/test/java/roomescape/MemberLogicTest.java

```diff
@@ -0,0 +1,99 @@
+package roomescape;
+
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.exception.CreateMemberFailException;
+import roomescape.member.dto.MemberRequest;
+
+
+@Transactional
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class MemberLogicTest {
+
+    @LocalServerPort
+    private int port;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    @Test
+    @DisplayName("아이디_생성_테스트")
+    void createMemberTest() {
+        // given
+        Map<String, String> memberRequest = new HashMap<>();
+        memberRequest.put("name", "Doyo");
+        memberRequest.put("email", "member@example.com");
+        memberRequest.put("password", "password");
+
+        // when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .body(memberRequest)
+                .contentType(ContentType.JSON)
+                .post("/members")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(201);
+        assertThat(response.jsonPath().getString("id")).isNotNull();
+        assertThat(response.jsonPath().getString("name")).isEqualTo("Doyo");
+        assertThat(response.jsonPath().getString("email")).isEqualTo("member@example.com");
```

Comment:
> RestAssured는 RESTful API 테스트를 위한 Java 라이브러리이고, API의 동작이 예상대로 수행되는지 검증할 때 사용되는데, 컨트롤러의 메서드를 호출하기 보다는 RestAssured를 사용할 수 있도록 통일했습니다.

말 그대로 RestAssured 는 테스트를 위해 사용하는 라이브러리입니다. 그런데, 검증부를 꼭 `Assertions.assertThat()`을 사용해야 할까요? 
아래와 같이도 사용이 가능하니 조금 더 찾아보셔도 좋을 것 같네요~
```suggestion
        RestAssured.given().log().all()
                .body(memberRequest)
                .contentType(ContentType.JSON)
                .post("/members")
            .then().log().all()
                .statusCode(201)
                .body("id", notNullValue())
                .body("name", is("Doyo"))
                .body("email", is("member@example.com"));
```